release-notes-3.0.0.md

```diff
@@ -3,6 +3,9 @@
 #### Version Number
 ${version-number}
 
+#### Breaking Changes
+- US975260: Refactor to remove HPE references in package names
```

Comment:
I don't think there are actually any breaking changes to this component.  Are there?

Reply:
Yeah there aren't. I will reversion it back to 2.2.0.